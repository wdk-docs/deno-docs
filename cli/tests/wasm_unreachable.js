// WebAssembly module containing a single function with an unreachable instruction
const binary = Uint8Array.from([
  0x00,
  0x61,
  0x73,
  0x6d,
  0x01,
  0x00,
  0x00,
  0x00,
  0x01,
  0x04,
  0x01,
  0x60,
  0x00,
  0x00,
  0x03,
  0x02,
  0x01,
  0x00,
  0x07,
  0x0f,
  0x01,
  0x0b,
  0x75,
  0x6e,
  0x72,
  0x65,
  0x61,
  0x63,
  0x68,
  0x61,
  0x62,
  0x6c,
  0x65,
  0x00,
  0x00,
  0x0a,
  0x05,
  0x01,
  0x03,
  0x00,
  0x00,
  0x0b,
]);

const module = new WebAssembly.Module(binary);
const instance = new WebAssembly.Instance(module);

instance.exports.unreachable();
