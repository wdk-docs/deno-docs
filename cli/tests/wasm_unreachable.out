error: Uncaught RuntimeError: unreachable
    at unreachable (<anonymous>:0:40)
    at [WILDCARD]/deno/cli/tests/wasm_unreachable.js:[WILDCARD]:18
